apis/r/man/soma_array_reader.Rd

```diff
@@ -54,9 +54,7 @@ not iterate; if your data is large than the initial read size consider the \code
 functions.
 }
 \examples{
-\dontrun{
```

Comment:
I think we are better off keeping these.  Our test coverage is good, this just slows things down and if we ever go to CRAN we have to inject throttles to 'Ncpu' cores in all examples...

Reply:
Ah okay, I'll revert. I just wanted to verify the examples worked.